Dockerfile

```diff
@@ -1,23 +1,23 @@
-FROM node:alpine
+FROM node:6-alpine
 
-ENV appdir /usr/local/app
+ENV APPDIR /usr/local/app
 
-RUN mkdir -p $appdir
-WORKDIR  $appdir
-COPY . $appdir
+RUN mkdir -p "$APPDIR"
+WORKDIR $APPDIR
+COPY . $APPDIR
 RUN npm install
 
```

Comment:
Could you add the MAINTAINER ? I guess the name of the project would be enough, unless :) you want to add your own email